coordinator/gscoordinator/cluster.py

```diff
@@ -908,10 +899,33 @@ def _create_interactive_engine_service(self):
             )
         )
 
+    def _config_etcd_endpoint(self):
+        if not self._etcd_addrs:
+            self._create_etcd()
+            self._etcd_endpoint = self._get_etcd_service_endpoint()
+            logger.info("Etcd is ready, endpoint is {}".format(self._etcd_endpoint))
+        else:
+            self._etcd_endpoint = self._etcd_addrs
+            logger.info("Etcd endpoint is {}".format(self._etcd_endpoint))
+
+    def _get_etcd_endpoints(self):
+        etcd_addrs = []
+        if not self._etcd_addrs:
+            port = self._random_etcd_listen_client_service_port
+            etcd_addrs.append("%s:%s" % (self._etcd_service_name, port))
+            for i in range(self._etcd_num_pods):
+                etcd_addrs.append("%s-%d:%s" % (self._etcd_name, i, port))
+        else:
+            for i in self._etcd_addrs.split(sep=","):
```

Comment:
You can just use  `etcd_addrs = self._etcd_addrs.split(",")` to simplify the code.